<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="NoRebootCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No Reboot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RebootDownloadButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Reboot Download</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="RebootButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Reboot</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QCheckBox" name="CSCCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CSC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CPPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="APOpenButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="APPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BLOpenButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BLPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="APCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>AP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CSCPath">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>291</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CPOpenButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BLCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CPCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CSCOpenButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>71</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>311</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
